providers/implementations/keymgmt/ec_kmgmt.c

```diff
@@ -745,9 +745,21 @@ int common_get_params(void *key, OSSL_PARAM params[], int sm2)
         }
         p->return_size = EC_POINT_point2oct(ecg, ecp,
                                             POINT_CONVERSION_UNCOMPRESSED,
-                                            p->data, p->return_size, bnctx);
-        if (p->return_size == 0)
+                                            NULL, 0, bnctx);
+        /* Check input buffer size */
+        if (p->return_size == 0
+            || (p->data != NULL && p->data_size < p->return_size)) {
+            /* Buffer too small or return size is 0, return error value 0. */
             goto err;
+        }
+
+        if (p->data != NULL) {
+            p->return_size = EC_POINT_point2oct(ecg, ecp,
+                                            POINT_CONVERSION_UNCOMPRESSED,
```

Comment:
fix the indent here and on the next line

Reply:
Ah, my bad